bin/migrate/migrate_ati_2019_11.py

```diff
@@ -18,8 +18,11 @@
         if 'warehouse' in sys.argv[1:]:
             line['user_modified'] = ''  # special "we don't know" value
         else:
-            raise KeyError('Invalid value')
+            line['user_modified'] = '*'
```

Comment:
need to move the comment above to this line, same for other changes

Reply:
if you're updating them all anyway I bet you can omit the `line['user_modified'] = ''` block because we don't seem to have it for the other new fields